content/applications/inventory_and_mrp/inventory/shipping/setup/sendcloud_shipping.rst

```diff
@@ -0,0 +1,227 @@
+==========================================
+Set up Sendcloud shipping services in Odoo
+==========================================
+
+Sendcloud is a shipping service aggregator. Sendcloud facilitates the integration of European
+shipping carriers with Odoo. Once integrated, the user can select the shipping carrier on inventory
+operations in their Odoo database.
+
+.. note::
+   Here is `Sendcloud's documentation <https://support.sendcloud.com/hc/en-us/articles
+   /360059470491-Odoo-integration>`_ on setting up the integration.
+
+.. important::
+   Odoo integration of Sendcloud does **not** work on free plans of Sendcloud.
+
+Setup in Sendcloud
+===================
+
+Create an account and activate carriers
+---------------------------------------
+
+To get started, go to `Sendcloud's platform <https://www.sendcloud.com>`_ to configure the account
+and generate the connector credentials. Log in with the Sendcloud account.
+
+If the user does not have a Sendcloud account, fill in the contact information and payment details.
+Sendcloud will ask for a :abbr:`VAT (Value-Added Tax Identification)` number or :abbr:`EORI
+(Economic Operators' Registration and Identification)` number. After completing the account setup,
+activate (or deactivate) the shipping carriers that will be used in the Odoo database.
+
+Warehouse configuration
+-----------------------
+
+Once the Sendcloud account is created, set up the :guilabel:`Warehouse Address`. Start in the
+Sendcloud account, go to :menuselection:`Settings --> Shipping --> Addresses`, and enter the
+:guilabel:`Warehouse address`.
+
+.. image:: sendcloud_shipping/settings-shipping.png
+   :align: center
+   :alt: Adding warehouse addresses in the Sendcloud settings.
+
+If the company would like Sendcloud to process returns as well, a :guilabel:`Return Address` is
+required. Under the :guilabel:`Miscellaneous section`, there is a field called :guilabel:`Address
+Name (optional)`. The Odoo warehouse name should be entered here, and the characters should be
+exactly the same.
+
+.. example::
+   | *An example of the configuration:*
+
+   | **Sendcloud configuration:**
+   | Miscellaneous
+   | Address Name (optional): **Warehouse #1**
+   | Brand: Default
+
+   | **Odoo warehouse configuration:**
+   | Warehouse: **Warehouse #1**
+   | Short Name: WH
+   | Company: My company (San Francisco)
+   | Address: My Company (San Francisco)
+
+   Notice how the name for both the Odoo configuration and the Sendcloud configuration are the exact
+   same.
+
+An example of the Sendcloud warehouse configuration:
+
+.. image:: sendcloud_shipping/warehouse-label.png
+   :align: center
+   :alt: Warehouse address name in the Sendcloud settings.
+
+An example of the Odoo warehouse configuration:
+
+.. image:: sendcloud_shipping/odoo-warehouse.png
+   :align: center
+   :alt: Warehouse settings in Odoo.
```

Comment:
The `example` tag and these images back to back are redundant so I'd suggest removing one of them. 

If you choose to keep this block over the `example`, then let's break the context dependency between the text and the images; the text should be able to stand on it's own without relying on images, as images are _supplemental_ to the written documentation.

Reply:
Noted, thanks. I've removed the images and left the example.